provision/minikube/keycloak/templates/keycloak.yaml

```diff
@@ -10,6 +10,19 @@ metadata:
   namespace: {{ .Values.namespace }}
 spec:
 # end::keycloak-ispn[]
+  {{- if .Values.stretchCluster }}
+  # evenly spread pods between zones and nodes.
+  scheduling:
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 50
+            podAffinityTerm:
+              topologyKey: topology.kubernetes.io/zone
```

Comment:
Should I add label selectors to match the Keycloak pods? It doesn't seem to be mandatory, but it sounds like a best practice.

Reply:
I think so, otherwise I believe the policy will also take into account non-keycloak pods which is not desirable.